tests/output/printers/LoggingTest.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+import logging
+import json
```

Comment:
though we don't have the bear ready yet, could you sort those imports in alphabetical order?